src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,17 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment }) => (
+  <>
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{comment.name}</strong>
```

Comment:
Use destruction assignment for comment object

Reply:
Not fixed